### ISSUE TITLE
Add PETOOH implementation

### ISSUE THREAD

**Issue by LavirtheWhiolet** [Ky6uk/PETOOH#33]
There are a bunch of interpreters of PETOOH written in Ruby, Python, Perl etc. but there is no interpreter written in PETOOH. This technique is called boostrapping and is very important for the modern information technologies and allows to leverage increasing complexities of the software development companies.

**Comment by Ky6uk** [comment]
Indeed. This is very important feature and should be implemented ASAP. We are will start working on it right now. Thanks!

**Comment by LavirtheWhiolet** [comment]
This issue critically depends on https://github.com/Ky6uk/PETOOH/issues/21.
